content/blog/2020-07-22-july-20-community-gems.md

```diff
@@ -0,0 +1,169 @@
+
+```dvc
+$ dvc repro <.dvc stage file>
+```
+
+substituting the appropriate `.dvc` file for your pipeline. DVC 1.0 is backwards
+compatible, so pipelines created with previous versions will still run.
+
+### [Q: Is there a way to setup my DVC remote so I can manually download files from it without going through DVC?](https://discordapp.com/channels/485586884165107732/563406153334128681/717458695709130764)
+
+When DVC adds a file to a remote repository (such as an S3 bucket, or an SSH
+file server), there's only one change happening: DVC calculates an md5 for the
+file and renames it with that md5. In technical terms, it's storing files in a
+"content-addressable way". That means if you know the hash of a file, you can
+locate it in your DVC remote and manually download it.
+
+To find the hash for a given file, say `data.csv`, you can look in the
+corresponding DVC file:
+
+```dvc
+$ cat data.csv.dvc
+```
+
+Another approach is using a built-in DVC function:
+
+```dvc
+$ dvc get --show-url . data.csv
+```
+
+You can read more about `dvc get --show-url` in
+[our docs](https://dvc.org/doc/command-reference/get#options). Note that this
+functinality is also part of our Python API, so you can locate the path to a
+file in your remote within a Python environment.
+[Check out our API docs!](https://dvc.org/doc/api-reference/get_url)
+
+## CML questions
+
+### Q: I use Bitbucket. Will CML work for me?
+
+The first release of CML is compatible with GitHub and GitLab. We've seen
+[many requests for Bitbucket support](https://github.com/iterative/cml/issues/140),
+and we're making it a development priority for an upcoming release. Stay tuned.
+
+### [Q: I have on-premise GPUs. Can CML use them to execute pipelines?](https://discordapp.com/channels/485586884165107732/728693131557732403/730070747388706867)
+
+Yep! You can use on-premise compute resources by configuring them as self-hosted
+runners. See
+[GitHub](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)
+and [GitLab](https://docs.gitlab.com/runner/)'s official docs for more details
+and setup instructions.
+
+### [Q: I'm building a workflow that deploys a GCP Compute Engine instance, but I can only find examples with AWS EC2 in the CML docs. What do I do?](https://discordapp.com/channels/485586884165107732/728693131557732403/730688592787275806)
+
+There is a slight difference in the way CML handles credentials for AWS and GCP,
+and that means you'll have to modify your workflow file slightly. We've added
+this example to our
+[project README](https://github.com/iterative/cml#allocating-cloud-resources-with-cml):
+
+```yaml
+deploy-gce:
+  runs-on: [ubuntu-latest]
+  container: docker://dvcorg/cml-cloud-runner
+
+  steps:
+    - name: deploy
+      shell: bash
+      env:
+        repo_token: ${{ secrets.REPO_TOKEN }}
+        GOOGLE_APPLICATION_CREDENTIALS_DATA:
+          ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS_DATA }}
+      run: |
+
```

Comment:
extra space? is it needed? 

looks a bit too much to copy/paste the whole yaml, let's link to the example (so that we update it in a single place), or gist

Reply:
Yep just linking to example.